### ISSUE TITLE
🐛 Bug: alt-text-bot-id causes incorrect formatting of link on its line

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [github/accessibility-alt-text-bot#62]
Looking at https://github.com/JoshuaKGoldberg/eslint-plugin-package-json/pull/721#issuecomment-2594302755:

> <div alt-text-bot-id="2594302487" /> Uh oh! _(username redacted to avoid ping)_, at least one image you shared is missing helpful alt text. Check https://github.com/[/pull/721](https://github.com/JoshuaKGoldberg/eslint-plugin-package-json/pull/721)#issuecomment-2594302487 to fix the following violations:
> * Images should have meaningful alternative text (alt text) at line 2
> 
> ...

Looking at the source:

> ```plaintext
> <div alt-text-bot-id="2594302487" /> ... Check https://github.com/JoshuaKGoldberg/eslint-plugin-package-json/pull/721#issuecomment-2594302487 to fix the following violations:
> ```

The _"Check ..."_ portion should render correctly on its own. But the `<div ... />` switches the line to parsing as HTML rather than Markdown, I think. So it's rendering the text as:

> ```plaintext
> Check https://github.com//pull/721#issuecomment-2594302487 to fix the following violations
> ```
